ts/torch_handler/base_handler.py

```diff
@@ -64,16 +62,16 @@ def initialize(self, context):
         if model_file:
             logger.debug("Loading eager model")
             self.model = self._load_pickled_model(model_dir, model_file, model_pt_path)
+            self.model.to(self.device)
```

Comment:
map_location (if done correctly) should have taken care of sending model to device

Reply:
@ngimel, I wonder if map_location moves the state_dict then model should be moved separately as well, as recommended [here](https://pytorch.org/tutorials/beginner/saving_loading_models.html#save-on-cpu-load-on-gpu)